students/BrianNisonger/session9/mailroom.py

```diff
@@ -0,0 +1,197 @@
+import statistics
+
+
+class Donor():
+    """ Class used to contain donor name,donation, and methods involved with a single donor """
+
+    def __init__(self, name, donation=""):
+        """ Takes a name, and optionally a donation list"""
+        self.name = name
+        self.donation = list(donation)
```

Comment:
the list constructor makes a list from an iterable.

if donation is an empty string (the default), then this will get you and empty string. but if it's a number or string with a number in it, you will not get what you expect!

```
In [9]: list("")                                                                
Out[9]: []

In [10]: list("500")                                                            
Out[10]: ['5', '0', '0']

In [11]: list(500)                                                              
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-11-76c72db8cca4> in <module>
----> 1 list(500)

TypeError: 'int' object is not iterable
```

Reply:
make sure you test creation of a donor with both nothing AND a donation, of it's an option to do either.

If you want to only create empty donors, that's fine, too, but then don't make it a parameter.